common/terms.html

```diff
@@ -1,343 +1,348 @@
-    it's values are merged with the map value when expanding.</dd>
-  <dt><dfn>JSON literal</dfn></dt><dd>
-    A <a>JSON literal</a> is a <a>literal</a> where the associated <a>IRI</a> is <code>rdf:JSON</code>.
-    In the <a>value object</a> representation, the value of <code>@type</code> is <code>@json</code>.
-    JSON literals represent values which are valid JSON [[RFC8259]].
-    See <dfn data-cite="JSON-LD11#dfn-json-datatype" class="preserve">JSON datatype</dfn> in [[JSON-LD11]].
-  </dd>
-  <dt><dfn class="preserve">typed value</dfn></dt><dd>
-    A <a>typed value</a> consists of a value,
-    which is a <a>string</a>,
-    and a type,
-    which is an <a>IRI</a>.</dd>
-  <dt><dfn>value object</dfn></dt><dd>
-    A <a>value object</a> is a <a>map</a> that has an <code>@value</code> <a>entry</a>.</dd>
-  <dt><dfn class="preserve">vocabulary mapping</dfn></dt><dd>
-    The vocabulary mapping is set in the <a>context</a> using the <code>@vocab</code> key
-    whose value MUST be an <a>IRI</a> or <code>null</code>.</dd>
-</dl>
+    <dt><dfn data-cite="INFRA#list" class="preserve">array</dfn></dt><dd>
+      In the JSON serialization,
+      an <a>array</a> structure is represented as square brackets surrounding zero or more values.
+      Values are separated by commas.
+      In the <a>internal representation</a>,
+      a <a data-lt="array">list</a> (also called an <a>array</a>) is an <em>ordered</em> collection of zero or more values.
+      While JSON-LD uses the same array representation as JSON,
+      the collection is <em>unordered</em> by default.
+      While order is preserved in regular JSON arrays,
+      it is not in regular JSON-LD arrays unless specifically defined
+      (see <a data-cite="JSON-LD11#sets-and-lists">Sets and Lists</a>
+      in the JSON-LD Syntax specification [[JSON-LD11]]).</dd>
+    <dt><dfn data-cite="RFC8259#section-4" class="preserve">JSON object</dfn></dt><dd>
+      In the JSON serialization,
+      an <a>object</a> structure
+      is represented as a pair of curly brackets surrounding zero or more name/value pairs (or members).
+      A name is a <a>string</a>.
+      A single colon comes after each name,
+      separating the name from the value.
+      A single comma separates a value from a following name.
+      In JSON-LD the names in an object MUST be unique.
+      <p>In the <a>internal representation</a> a <a>JSON object</a> is described as a
+        <dfn data-cite="INFRA#ordered-map" class="preserve">map</dfn> (see [[INFRA]]),
+        composed of <dfn data-cite="INFRA#map-entry" data-lt="map entry|entry" data-lt-noDefault class="preserve">entries</dfn> with key/value pairs.</p>
+      <p>In the <a data-cite="JSON-LD11-API#the-application-programming-interface">Application Programming Interface</a>,
+        a <a>map</a> is described using a [[WEBIDL]] <a>dictionary</a>.</p></dd>
+    <dt class="changed">JSON-LD <a>internal representation</a></dt>
```

Comment:
Here is an example of xref auto-linking a common term.

(I think this document has Window's line endings, which Visual Studio Code removed)